### PR TITLE
Add docker, docker-compose and GitHub Actions support for Podiscover

### PR DESCRIPTION
**Pull request by kiliczsh**
-  rspotify gem has improvements and served from https://gem.fury.io/enderahmetyurt/ replaced with default rspotify which needs to be fixed.
- faker gem should be update with main branch instead of master.
- README.md file has two callbacks links but it may need further configuration to make the app work.
- Tailwind installation may be pruned but right now it is working.
- Action added for Pull Request to avoid compilation errors.

### CODE REVIEW

**Comment by enderahmetyurt** [comment]
It's very nice Thank you very much ❤️ I'll review it asap 🤓 @kiliczsh

File: Gemfile.docker [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+# source 'https://gem.fury.io/enderahmetyurt/' do
+#   gem 'rspotify'
+# end
```
Comment (enderahmetyurt) [inline Gemfile.docker:8]: we can remove this instead commenting it

File: Gemfile.docker [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+gem "sentry-ruby"
+gem "sentry-rails"
+
+gem 'flipper'
+gem 'flipper-active_record'
+
+gem "view_component"
+gem 'active_analytics'
+gem 'rspotify'
```
Comment (enderahmetyurt) [inline Gemfile.docker:112]:
```suggestion
gem 'rspotify', git: 'https://github.com/enderahmetyurt/rspotify'
```
could you please try this?

File: README.md [hunk 3]
```diff
@@ -3,3 +3,43 @@
 Find your next podcast.
 
 www.podiscover.me
```
Comment (enderahmetyurt) [inline README.md:5]: we can remove this

**Review by enderahmetyurt — CHANGES_REQUESTED** [review]

File: Gemfile.docker (hunk 1 above)
Comment (kiliczsh) [reply Gemfile.docker]: removed Gemfile.docker file

File: Gemfile.docker (hunk 2 above)
Comment (kiliczsh) [reply Gemfile.docker]: Thanks it worked for rspotify gem 💎

File: README.md (hunk 3 above)
Comment (kiliczsh) [reply README.md]: removed 👍🏼

**Review by enderahmetyurt — APPROVED** [review]

**Pull request merged**